manifests/config.pp

```diff
@@ -98,11 +98,13 @@
     )
   )
   $min_puma_threads = pick($foreman::foreman_service_puma_threads_min, $foreman::foreman_service_puma_threads_max)
-  systemd::dropin_file { 'foreman-service':
-    filename       => 'installer.conf',
-    unit           => "${foreman::foreman_service}.service",
-    content        => template('foreman/foreman.service-overrides.erb'),
-    notify_service => true,
+  if $foreman::deployment_mode == 'package' {
```

Comment:
Does it makes sense to make the deployment mode ensure absent/present?

Reply:
Yes, that's my next step :)